### ISSUE TITLE
When I try to run server.py I get this error

### ISSUE THREAD

**Issue by prathameshparit** [Zheng-Chong/FashionMatrix#3]
ERROR:    Exception in ASGI application
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/uvicorn/protocols/http/h11_impl.py", line 408, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
  File "/usr/local/lib/python3.10/dist-packages/uvicorn/middleware/proxy_headers.py", line 84, in __call__
    return await self.app(scope, receive, send)
  File "/usr/local/lib/python3.10/dist-packages/fastapi/applications.py", line 289, in __call__
    await super().__call__(scope, receive, send)
  File "/usr/local/lib/python3.10/dist-packages/starlette/applications.py", line 122, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/errors.py", line 184, in __call__
    raise exc
  File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/errors.py", line 162, in __call__
    await self.app(scope, receive, _send)
  File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/exceptions.py", line 79, in __call__
    raise exc
  File "/usr/local/lib/python3.10/dist-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/usr/local/lib/python3.10/dist-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/local/lib/python3.10/dist-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.10/dist-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/usr/local/lib/python3.10/dist-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/usr/local/lib/python3.10/dist-packages/starlette/routing.py", line 66, in app
    response = await func(request)
  File "/usr/local/lib/python3.10/dist-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
  File "/usr/local/lib/python3.10/dist-packages/fastapi/routing.py", line 190, in run_endpoint_function
    return await dependant.call(**values)
  File "/content/FashionMatrix/server.py", line 64, in graph
    response = model_graph.process(image_path=image_path)
NameError: name 'model_graph' is not defined

**Comment by prathameshparit** [comment]
And this error for app_label.py

```
Running on local URL:  http://127.0.0.1:7860
Running on public URL: https://d1d7515c53535aa249.gradio.live

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run `gradio deploy` from Terminal to deploy to Spaces (https://huggingface.co/spaces)
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/requests/models.py", line 971, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/gradio/routes.py", line 442, in run_predict
    output = await app.get_blocks().process_api(
  File "/usr/local/lib/python3.10/dist-packages/gradio/blocks.py", line 1392, in process_api
    result = await self.call_function(
  File "/usr/local/lib/python3.10/dist-packages/gradio/blocks.py", line 1097, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/usr/local/lib/python3.10/dist-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/usr/local/lib/python3.10/dist-packages/anyio/_backends/_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "/usr/local/lib/python3.10/dist-packages/anyio/_backends/_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "/usr/local/lib/python3.10/dist-packages/gradio/utils.py", line 703, in wrapper
    response = f(*args, **kwargs)
  File "/content/FashionMatrix/app_label.py", line 77, in image_change
    parts = FA.existing_parts(img_path)
  File "/content/FashionMatrix/roles/assistant.py", line 93, in existing_parts
    return self.FD.existing_parts(img_path)
  File "/content/FashionMatrix/roles/designer.py", line 118, in existing_parts
    graph = self.AM.co_segm(image_path)
  File "/content/FashionMatrix/roles/automasker.py", line 161, in co_segm
    graph = self.graph_dict(img_path)
  File "/content/FashionMatrix/roles/automasker.py", line 113, in graph_dict
    graph = self.graph_np(img_path)
  File "/content/FashionMatrix/roles/automasker.py", line 99, in graph_np
    graph_path = self.hpm_api.graph(image_path=resized_path)
  File "/content/FashionMatrix/api.py", line 36, in graph
    response = self.post('graph', {'image_path': image_path})
  File "/content/FashionMatrix/api.py", line 10, in post
    return requests.post(self.prefix + endpoint, json=data).json()
  File "/usr/local/lib/python3.10/dist-packages/requests/models.py", line 975, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by prathameshparit** [comment]
Also in server.log this is what i get:

```
nohup: ignoring input
2023-08-13 10:17:15.557673: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
/content/FashionMatrix/models/vfm.py:10: FutureWarning: Importing `StableDiffusionControlNetPipeline` or `MultiControlNetModel` from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion_controlnet is deprecated. Please import `from diffusers import StableDiffusionControlNetPipeline` instead.
  from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion_controlnet import MultiControlNetModel
Traceback (most recent call last):
  File "/content/FashionMatrix/server.py", line 114, in <module>
    model_cn = MultiControlNet(controlnet_folder=os.path.join(args.checkpoints, 'ControlNet'), device="cuda")
  File "/content/FashionMatrix/models/vfm.py", line 104, in __init__
    self.pipe = StableDiffusionControlNetInpaintPipeline.from_pretrained(
  File "/usr/local/lib/python3.10/dist-packages/diffusers/pipelines/pipeline_utils.py", line 882, in from_pretrained
    cached_folder = cls.download(
  File "/usr/local/lib/python3.10/dist-packages/diffusers/pipelines/pipeline_utils.py", line 1185, in download
    info = model_info(
  File "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_validators.py", line 110, in _inner_fn
    validate_repo_id(arg_value)
  File "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_validators.py", line 158, in validate_repo_id
    raise HFValidationError(
huggingface_hub.utils._validators.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name': './checkpoints/realisticVisionV50_v50VAE'. Use `repo_type` argument if needed.
```

**Comment by Zheng-Chong** [comment]
This may be caused by not downloading the checkpoints of Graphonomy and RealisticVision to the specified location.

Realistic Vision V5.0 finetuned from SD-v1.5 can be downloaded from [CivitAI](https://civitai.com/models/4201?modelVersionId=125411). After downloading the safetensor file, you must convert it to diffusers format following instructions [here](https://github.com/haofanwang/Lora-for-Diffusers#full-models).

Graphonomy Model can be downloaded from:
https://drive.google.com/file/d/1eUe18HoH05p0yFUd_sN6GXdTj82aW0m9/view

And make sure all checkpoints in the folder shown in [INSTALL.md](https://github.com/Zheng-Chong/FashionMatrix/blob/main/INSTALL.md).

**Comment by Zheng-Chong** [comment]
But this version may have other problems, the next more user-friendly version v1.2 (less memory, optimized effects, easier deployment) will be released soon.

**Comment by Zheng-Chong** [comment]
We have released a new version 2.0, which uses less base model (thus requires less video memory and runs faster), please follow the installation guide to deploy, hope it can solve your problem.
